### ISSUE TITLE
`pybritive ls profiles -f json` lacks profile name

### ISSUE THREAD

**Issue by michaelbrewer** [britive/python-cli#141]
When listing accounts to be checked out `pybritive ls profiles -f json` does not include the profile name, which would later on be used for checkout

```json
[
  {
    "Application": "AWS",
    "Environment": "111111111111111111 (name-of-env-prod)",
    "Profile": "111111111111111111-role-name",
    "Description": "XXXXXXXXX.",
    "Type": "AWS"
  },
]
```

Where as npm brivitve cli used to offer this

britive listprofiles -f json
```json
[
    {
        "application": "AWS",
        "environmentName": "111111111111111111 (name-of-env-prod)",
        "name": "AWS/name-of-env-prod/111111111111111111-role-name",
        "description": "XXXXXXXXX.",
        "type": "AWS"
    },
]
```

Then you can use the `name` field to checkout the environment

```bash
name=AWS/name-of-env-prod/111111111111111111-role-name
britive checkout -r us-east-2 -a profileName $name -s
```

> NOTE: Although a long form name is supported if you combine "$Application/$Environment/$Profile"
     `pybritive checkout -a profile "AWS/111111111111111111 (name-of-env-prod)/111111111111111111-role-name" -m integrate`

**Comment by twratl** [comment]
That information can be found with `pybritive ls profiles -f list` which makes it easy to copy/paste or filter further via `grep` and similar tools. Happy to explore adding it as another field in the `-f json` output.

**Comment by michaelbrewer** [comment]
Yes. Then we can easily script things together in a chain. 

The idea is that a user can select different  accounts via a terminal interface, check then out and store them as profiles and then use these to bootstrap something across multiple accounts. 

Maybe all of this is better suited for the python SDK. As long as it can also launch the web browser. 

Regardless, the old method works with minimal code. It would have helped test this cli tool was a drop in replacement.

**Comment by twratl** [comment]
The CLI offers an AWS `credential_process` option which would allow you to define whatever profiles up front that you want and have the AWS CLI/SDK call `pybritive` whenever it needs credentials. These credentials would be cached locally on your machine in an encrypted format in an effort to reduce network calls. When they expire the system would take care of checking out new credentials automatically, and pop a browser for re-authentication, if required.

In the example below I have AWS profiles `a,b,c, and d` and just also created Britive profile aliases of the same names. `pybritive configure update profile-aliases friendy-name "britive-name"` and/or you can set an alias during checkout command with `--alias/-a`.

~~~
[a]
credential_process=pybritive checkout a -m awscredentialprocess
region=us-east-1

[b]
credential_process=pybritive checkout b -m awscredentialprocess
region=us-east-1

[c]
credential_process=pybritive checkout c -m awscredentialprocess
region=us-west-2

[d]
credential_process=pybritive checkout d -m awscredentialprocess
region=us-west-2
~~~

With this you start your work day directly using the AWS CLI (an an example). 

`aws s3 ls --profile a` would pop a browser for auth, checkout the credentials, and store them locally. When those credentials expire it would start over again.

**Comment by michaelbrewer** [comment]
@twratl  - i have created a PR to add name for the json output only

**Comment by twratl** [comment]
This has been released in v1.7.0, which is now available.
